examples/prefetch/producer.js

```diff
@@ -0,0 +1,33 @@
+
+var producer = require('../../index')().producer;
+
+var i = 0;
+var interval;
+
+interval = setInterval(function () {
+  producer.produce('queue-prefetch' + i, { message: 'start-' + i }, { rpc: true })
+  .then(function (result) {
+    console.log('result:', result);
+  });
+
+  ++i;
+  if (i >= 100) {
+    interval = clearInterval(interval);
+  }
+}, 500);
+
+// producer.connect()
```

Comment:
@ky23  Hey are the commented out sections indicative of something (example) or they can be removed maybe ?